### PR TITLE
Feature/max edge segment

### PR DESCRIPTION
**Pull request by steviedale**

### CODE REVIEW

File: tool_path_planner/src/utilities.cpp [hunk 1]
```diff
@@ -298,4 +299,48 @@ bool createToolPathSegment(const pcl::PointCloud<pcl::PointNormal>& cloud_normal
...
+// this is a helper function for splitPaths()
+double getDistance(Eigen::Isometry3d t1, Eigen::Isometry3d t2)
+{
+  auto p1 = t1.translation();
+  auto p2 = t2.translation();
+  return sqrt(pow(p1.x() - p2.x(), 2) + pow(p1.y() - p2.y(), 2) + pow(p1.z() - p2.z(), 2));
+}
+
+ToolPaths splitSegments(ToolPaths tool_paths, double max_segment_length)
+{
```
Comment (drchrislewis) [inline tool_path_planner/src/utilities.cpp:311]: We need to have some mechanism to bypass splitSegments(). For example, we could set max_segment_length to 0 or even negative, if so, have this routine do nothing.

Comment (steviedale) [reply tool_path_planner/src/utilities.cpp:311]: Addressed in 12181b3

**Comment by Levi-Armstrong** [comment]
@steviedale Can you run clang formatting?

**Comment by Levi-Armstrong** [comment]
There is a `.run-clang-format` so you in the terminal you can run it using `./.run-clang-format`.

**Comment by Levi-Armstrong** [comment]
@DavidMerzJr This ready to be merged?

**Comment by DavidMerzJr** [comment]
Included in !2

**Pull request closed without merging**